conductor/submitter_maya.py

```diff
@@ -258,6 +292,8 @@ def generateTasksData(self):
         # in the maya scene will be used).
         else:
             renderer_arg = ""
+                    
+        renderer_arg += self.extended_widget.get_render_log_arg()            
```

Comment:
I'm not sure if this is technically a problem (since I'm assuming you've tested it), but by appending (concatenating) the new log flag/arg to the existing `renderer_arg` variable, you're going to end up with a Render command that provides an arnold-specific rendering flag (`ai:lve`), but will not have explicitly specified arnold as the renderer to use (since the `renderer_arg` is always an empty string, except when using renderman).  I would have guessed (hoped) that maya would reject a command if it's been provided a renderer-specific argument but no renderer has been specified.

Reply:
I think, even if Arnold is the renderer set inside the scene file, you can't specify `ai:lve` on the command line unless you also specify `-renderer ai`. I could be wrong, but I was trying to debug a customer scene by changing the command in datastore the other day and it failed when I just added the flag to a command that didn't specify the renderer.

Reply:
@lawschlosser  `get_render_log_arg()` only returns a valid string for the current renderer. At the moment, it will be empty if Arnold isn't selected.

@hoolymama I've never run into that... though I just ran a test and you're right. I'll push changes to ensure that doesn't happen